src/AppBundle/Entity/Container.php

```diff
@@ -378,6 +387,28 @@ public function removeBackupSchedule(BackupSchedule $backupSchedule){
         $backupSchedule->removeContainer($this);
     }
 
+    /**
+     * @param Backup $backup
+     */
+    public function addBackup(Backup $backup){
+        if ($this->backups->contains($backup)) {
+            return;
+        }
+        $this->backups->add($backup);
+        $backup->addContainer($this);
+    }
+
+    /**
+     * @param Backup $backup
+     */
+    public function removeBackup(Backup $backup){
+        if (!$this->backups->contains($backup)) {
+            return;
+        }
+        $this->backups->removeElement($backup);
+        $backup->removeContainer($this);
+    }
+
```

Comment:
Bräuchten wir hier nicht auch noch irgendwie `getBackups()`?

Reply:
Stimmt und getContainers() in der Backup entity, füge ich heute abend hinzu